src/Approvals/SFA.DAS.Approvals/Services/ProviderCoursesService.cs

```diff
@@ -98,12 +104,26 @@ private async Task<IEnumerable<Standard>> GetAllStandards()
 
         private async Task<IEnumerable<Standard>> GetStandardsForProvider(long providerId)
         {
-            var providerStandards =
-                await _providerCoursesApiClient.Get<IEnumerable<GetProviderStandardsResponse>>(
-                    new GetProviderStandardsRequest(providerId));
+            try
+            {
+                var providerStandards =
```

Comment:
We could .ToList() the response, to prevent possible multiple enumeration of the IEnumerable.

Reply:
Done